### DISCUSSION TITLE
Two Seq instances on Kubernetes using shared configuration in MS-SQL

### DISCUSSION THREAD

**Discussion by kellybirr** [datalust/seq-tickets#1825] [Help] [answered]
I have two production AKS clusters where my apps are deployed active/active.  I have a local Seq instance installed in each cluster.  Both instances use Azure SQL as their metadata store (currently seperate).

Can these two instances use the same SQL database for configuration to share the configuration?  This way users, signals, queries, alerts, etc... configured one exist on the other automatically?

I don't care about sharing or replicating events, I know that's a "not yet".  My users can query both servers for now.  I would like to help them not have to duplicate their setups.

**Comment by nblumhardt** [comment, marked as answer]
G'day Kelly, thanks for the note. If you connect the two instances in a DR pair, they'll be able to share the backing SQL database. Logs will also be replicated, however.

Otherwise, no, there's no means for two standalone nodes to share configuration this way.

It sounds like using a single Seq instance in one cluster, and having apps in the other cluster log to it, would be the most pleasant experience, but you could also investigate using `seqcli` templates to migrate important signals or dashboards backwards and forwards between the two systems:

https://blog.datalust.co/copying-signals-and-dashboards-using-seqcli-templates/

Hope this helps,
Nick

**Reply by kellybirr** [reply]
Thanks, I appreciate the quick response.  A single Seq instance would be nice, but it would be way too much cross-cluster traffic and if that cluster failed, we'd lose Seq for everything.  So alas, we'll keep doing double configs for now.

Using SeqCli for anything is not a good option for us.  These instances are fully containerized in Kubernetes.  We'd have to stop the Seq Pod and then deploy a custom Job in Kubernetes for every command we'd want to execute.

**Reply by nblumhardt** [reply]
Thanks for your reply! I didn't mean that `seqcli` would be used in-cluster, only that on a machine with access to both clusters, dashboards etc. could be synchronized by pulling from one and pushing to the other (anyone with access to both Seq instances via HTTP would have sufficient connectivity to do this). HTH!

**Reply by kellybirr** [reply]
I didn't think of that.  I'll play with that some.

Thanks again!
